inc/ZipProvider.php

```diff
@@ -127,6 +127,31 @@ public function generate_zip_file() : bool {
 			return false;
 		}
 
+		// Build a mapping based on where the translation entries occur and separate the po entries.
+		$mapping    = $this->build_mapping( $entries );
+		$po_entries = array_key_exists( 'po', $mapping ) ? $mapping['po'] : [];
```

Comment:
Maybe rename this to `$php_entries`?

Reply:
Likewise, what about using `$all_entries` instead as it will effectively contain all entries?

Reply:
Does it need to contain all entries? Since we only have one project we don't seem to have the same issue as the core project has.

Reply:
That would help, yes.

Then we can remove the

```php
// Always add all entries to the PO file.
$sources[] = 'po';
```

part from `\Required\Traduttore\Export::map_entries_to_source()` as well.

After that, `$php_entries` instead of `$po_entries` is easier to understand.